pkg/collector/container_accelerator_collector.go

```diff
@@ -53,7 +54,14 @@ func (c *Collector) updateAcceleratorMetrics() {
 				c.createProcessMetricsIfNotExist(uint64(pid), "")
 			}
 
-			// update container metrics
+			// When a container or pod is deleted, the associated process data may still exist for some time. However, the createContainersMetricsIfNotExist function
+			// will not create an entry in the c.ContainersMetrics map if the container or pod no longer exists. Therefore, even if the function attempted to initialize
+			// the map with the deleted containerID, it may not have been added to the map due to the absence of container information. As a result, these processes
+			// may be classified as system processes rather than being attributed to the deleted container or pod.
+			if _, exist := c.ContainersMetrics[containerID]; !exist {
+				continue
+			}
+			klog.Infoln("DEBUG: pod", c.ContainersMetrics[containerID].PodName, "processUtilizationSample", processUtilization)
```

Comment:
can this be moved to V(5)?

Reply:
why Infoln with debug msg?

Reply:
Sorry this was for my own debugging...and should be removed